ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1152,7 +1152,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_ChatAndNotificationsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
```

Comment:
If you want to do right alignment, you may need to use a dock panel instead of a stack panel.